tsparser/src/builder/codegen.rs

```diff
@@ -159,6 +163,18 @@ impl Builder<'_> {
                 let mut endpoint_ctx = Vec::new();
                 let mut subscription_ctx = Vec::new();
 
+                let service_ctx = service.map(|service| {
```

Comment:
I was assuming that we only wanted to support middewares from encore.service.ts? was that wrong? the `service_ctx` is only used so that we can import the service in the handlebars templates, so that we can access the middlewares from there

Reply:
Yeah we will only have middleware when there is a service object in `encore.service.ts`. But I was just thinking logically there is still a service, even if there is no `encore.service.ts`. But can change that later if we need it

Reply:
ah, I misunderstood what you meant, I've changed to always set it, but only set `import_path` if there is a `encore.service.ts`